src/Microsoft.TemplateEngine.Edge/FilterableTemplateInfo.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace Microsoft.TemplateEngine.Edge
 {
+    [Obsolete("The class is deprecated.")]
```

Comment:
Maybe `This` instead of `The`?

Reply:
I believe `the` in  this context means `this`, but I can change if it makes it more clear.